index.html

```diff
@@ -0,0 +1,177 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header id="page-header">
+        <a href="" id="my-logo" class="blue-text">My logo</a>
+        <nav>
+            <img src="icons/Hamburger-menu.png" alt="">
+        </nav>
+    </header>
+    <section id="section-1" class="portfolio-summary portfolio-section">
+        <h1 id="main-head" class="title">I am Salomon Glad to see you</h1>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work
+            and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="blue-text" class="connect-link">LET’S CONNECT</a>
+        <ul class="contact-list">
+            <li>
+                <a href="#">
+                    <img src="icons/twitter-icon.png" class="contact-list-icon">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="icons/Linkedin-icon.png" class="contact-list-icon">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="icons/m-icon.png" class="contact-list-icon">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="icons/github-icon.png" class="contact-list-icon">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="icons/Vector-icon.png" class="contact-list-icon">
+                </a>
+            </li>
+        </ul>
+    </section>
+    <section id="section-2" class="portfolio-section">
+        <div class="work">
+            <div class="work-image-container work-image-1"></div>
+            <h2 class="work-name title">Tonic</h2>
+            <h3 class="small-description">CANOPY <span class="small-description-highlight"> <img src="icons/Counter.png"
+                        class="counter"> Back End Dev <img src="icons/Counter.png" class="counter"> 2015</span></h3>
+            <p class="work-summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-used-tools">
+                <li class="tool">html</li>
+                <li class="tool">css</li>
+                <li class="tool">javascript</li>
+            </ul>
+            <button class="portfolio-btn">See Project</button>
```

Comment:
- [ Required ] Please consider adding button styling for the different button states `Active`, `Hover` and `Pressed` .  Please refer to this color palate on the Figma design for the exact color to use for different states. Also see the code snippet below for guidance on this.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/212343636-22f3cdb9-f111-44b0-9607-ebabbd06596a.png" />

```CSS
.portfolio-btn:hover {
   color: #fff;
   background-color: #6070ff;
}

.portfolio-btn:active {
    color: #fff;
    background-color: #2230d2;
}
```